interface/src/main.cpp

```diff
@@ -66,59 +66,215 @@ int main(int argc, const char* argv[]) {
+    QCommandLineOption urlOption(
+        "url",
+        "Start at specified URL location.",
+        "string"
+    );
+    QCommandLineOption protocolVersionOption(
+        "protocolVersion",
+        "Writes the protocol version base64 signature to a file?",
+        "path" // Why??
+    );
+    QCommandLineOption noUpdaterOption(
+        "no-updater",
+        "Do not show auto-updater."
+    );
+    QCommandLineOption checkMinSpecOption(
+        "checkMinSpec",
+        "Check if machine meets minimum specifications. The program will run if check passes."
+    );
+    QCommandLineOption runServerOption(
+        "runServer",
+        "Run the server."
+    );
+    QCommandLineOption listenPortOption(
+        "listenPort",
+        "Port to listen on."
+    );
+    QCommandLineOption serverContentPathOption(
+        "serverContentPath",
+        "Path to find server content.", // What content??
+        "serverContentPath"
+    ); // This data type will not be familiar to users.
+    QCommandLineOption overrideAppLocalDataPathOption(
+        "cache",
+        "Set test cache.",
+        "dir"
+    );
+    QCommandLineOption scriptsOption(
+        "scripts",
+        "Set path for defaultScripts. These are probably scripts that run automatically.",
+        "dir"
+    );
+    QCommandLineOption allowMultipleInstancesOption(
+        "allowMultipleInstances",
+        "Allow multiple instances to run."
+    );
+    QCommandLineOption displaysOption(
+        "display",
+        "Preferred display.",
+        "displays"
+    );
+    QCommandLineOption disableDisplaysOption(
+        "disable-displays",
+        "Displays to disable.",
+        "string"
+    );
+    QCommandLineOption disableInputsOption(
+        "disable-inputs",
+        "Inputs to disable.",
+        "string"
+    );
+    QCommandLineOption suppressSettingsResetOption(
+        "suppress-settings-reset",
+        "Suppress the prompt to reset interface settings."
+    );
+    QCommandLineOption oculusStoreOption(
+        "oculus-store",
+        "Let the Oculus plugin know if interface was run from the Oculus Store."
+    );
+    QCommandLineOption standaloneOption(
+        "standalone",
+        "Emulate a standalone device."
+    );
+    QCommandLineOption disableWatchdogOption(
+        "disableWatchdog",
+        "Disable the watchdog thread. The interface will crash on deadlocks."
+    );
+    QCommandLineOption systemCursorOption(
+        "system-cursor",
+        "Probably prevents changing the cursor when application has focus."
+    );
+    QCommandLineOption concurrentDownloadsOption(
+        "concurrent-downloads",
+        "Maximum concurrent resource downloads. Default is 16, except for Android where it is 4."
+    );
+    QCommandLineOption avatarURLOption(
+        "avatarURL",
+        "Override the avatar U.R.L."
+    );
+    QCommandLineOption replaceAvatarURLOption(
+        "replace-avatar-url",
```

Comment:
Missing value name here too

Reply:
I don't really understand why there are two avatar url parameters but I assume they both accept url strings.

Reply:
I looked and have no idea why either. Probably something that made sense for HiFi? Like there's one place that may be using the avatar as an argument, and another that needs to be able to override that. Almost certainly not important for us until we develop such a need.